### PR TITLE
async-and-await and asynchronous-code: moving Scrimba from Lesson to Additional Resources

### PR DESCRIPTION
**Pull request by ht-l1**
## Because 

To move the existing Scrimba embeds in lessons to Additional Resources as link text in corresponds to layout style guide.

## This PR

Moving the Scrimba in [Async and await](https://www.theodinproject.com/lessons/node-path-javascript-async-and-await) from Lesson position to Additional Resources.
Moving the Scrimba in [Asynchronous code](https://www.theodinproject.com/lessons/node-path-javascript-asynchronous-code) from Lesson position to Additional Resources.

## Issue

Related to #26057

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by ht-l1** [comment]
hello @thatblindgeye this my pull request related to Issue26057. I could not figure out how to assign the reviewer to your queue so I'm tagging you here :)

File: javascript/asynchronous_javascript_and_apis/async_and_await.md [hunk 1]
```diff
@@ -234,3 +230,4 @@ This section contains helpful links to other content. It isn't required, so cons
 
 1. This [Change promises to async/await video](https://www.youtube.com/watch?v=COKdtOgopWQ) is an example of how you can change callbacks, to promises, to async/await.
 2. This [Promises, Async and Await video](https://www.youtube.com/watch?v=vn3tm0quoqE) gives a comprehensive view of Promises, async, and await.
+3. For a more interactive explanation and example, try [Scrim](https://scrimba.com/scrim/crd4eMc6?embed=odin,mini-header,no-next-up) (let us know what you think of these).
```
Comment (thatblindgeye) [inline javascript/asynchronous_javascript_and_apis/async_and_await.md:233]:
```suggestion
3. For a more interactive explanation and example, try a [Scrim on async and await](https://scrimba.com/scrim/crd4eMc6?embed=odin,mini-header,no-next-up).
```

File: javascript/asynchronous_javascript_and_apis/asynchronous_code.md [hunk 2]
```diff
@@ -100,3 +96,4 @@ This section contains helpful links to other content. It isn't required, so cons
 2. The [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) for Promises.  It might not be the best resource for _learning_ all about them, but once you've read a more friendly article or tutorial, this will probably be the place you return to for a refresher.
 3. This [ES6 Promises video](https://www.youtube.com/watch?v=vQ3MoXnKfuQ) and [Promises by The Net Ninja](https://www.youtube.com/watch?v=yswb4SkDoj0) are both nice introductions to Promises if you need more repetition.
 4. This [Understanding JavaScript Promises Tutorial](https://www.digitalocean.com/community/tutorials/understanding-javascript-promises) is another good introduction.
+5. For a more interactive explanation and example, try [Scrim](https://scrimba.com/scrim/cof4e4fb797a2d0a236ea38ce?embed=odin,mini-header,no-next-up) (let us know what you think of these).
```
Comment (thatblindgeye) [inline javascript/asynchronous_javascript_and_apis/asynchronous_code.md:99]:
```suggestion
5. For a more interactive explanation and example, try a [Scrim on asynchronous coding](https://scrimba.com/scrim/cof4e4fb797a2d0a236ea38ce?embed=odin,mini-header,no-next-up).
```

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Thanks for the updates! Just had a couple of slight tweaks to the link text below

**Comment by ht-l1** [comment]
thanks @thatblindgeye ! I just committed the new changes based on your update

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**